lib/jblond/Diff/Renderer/Text/UnifiedCli.php

```diff
@@ -49,10 +49,7 @@ public function render(): string
         if (!isset($this->options['cliColor'])) {
```

Comment:
`$this->options['cliColor']` should be set by MainRendererAbstract.
Therefor this line will always evaluate to true.

Please also check remaining code which uses this variable.

Reply:
It is not always true. I check that via the example/cli.php script.

Reply:
That's why I'm saying 'should' while it's not. 😉
When extending `MainRendererAbstract`, the options for a sub renderer are supposed to be a merge of: 

- `MainRendererAbstract::$mainOptions` (Default renderer options)
- `SubRenderer::$subOptions` (Default child renderer options)
- The array parameter of `SubRenderer::__construct()` (custom options).

`SubRenderer::__construct()` should look something like:
```php
    public function __construct(array $options = [])
    {
        parent::__construct();
        $this->setOptions($this->subOptions); // Override/Add options MainRenderer with defaults of SubRenderer.
        $this->setOptions($options); // Override/Add options MainRenderer with customs.
    }
```
Method `setOptions`, inherited from `MainRendererAbstract`, will take care of the merges.

Currently `UnifiedCli::__construct()` is different:
```php
    public function __construct(array $options = [])
    {
        parent::__construct($options);
        $this->colors = new CliColors();
        $this->options = $options;
    }
````

You can look at `InlineCli.php` as an example on how to apply options.
Also it's described at https://github.com/JBlond/php-diff/wiki/b)-Parameters-and-Options#parameters-1
and https://github.com/JBlond/php-diff/wiki/b)-Parameters-and-Options#text-unified-cli-renderer